src/main/resources/META-INF/rewrite/junit5.yml

```diff
@@ -246,7 +253,7 @@ recipeList:
       groupId: io.vertx
       artifactId: vertx-junit5
       version: 4.x
-      onlyIfUsing: io.vertx.junit5.VertxExtension
+      onlyIfUsing: org.vertx.testtools.VertxUnitRunner
```

Comment:
"org.vertx.testtools.VertxUnitRunner" seems to be non-existent. Google search shows it only as a part of openrewrite docs
![image](https://github.com/user-attachments/assets/de05caab-87f8-4ac8-aa16-5b1138bc8dcb)

Reply:
Thanks a lot! I suspect that might have been botched at some point. I've now looked up the corresponding classes and have arrived at: https://github.com/openrewrite/rewrite-testing-frameworks/commit/2a3d615dc1eb1e70fb9386c502e10362b2b63982